src/dao/pages/UpdateDAODetails/UpdateDAODetails.tsx

```diff
@@ -6,18 +6,20 @@ import { Wizard, Color, LoadingDialog } from "@shared/ui-kit";
 import { WarningIcon } from "@chakra-ui/icons";
 import { SettingsForm } from "./types";
 import { useHandleTransactionSuccess } from "@dex/hooks";
-import { useDAOs, useUpdateDAODetails } from "@dao/hooks";
+import { useContract, useDAOs, useUpdateDAODetails } from "@dao/hooks";
 import { getDAOLinksRecordArray } from "../utils";
 import { isEmpty } from "ramda";
 
 export function UpdateDAODetails() {
   const { accountId: daoAccountId = "" } = useParams();
+  const daoAccountIdQueryResults = useContract(daoAccountId);
```

Comment:
If `accountId` / `daoAccountId` is an EVM address and no longer an Account Id, appropriately rename the variable to explicitly highlight this change.

Reply:
accountId is the EVMAddress for MultiSig but for FT and NFT DAOs it is AccountId. so with this PR I have not changed the name of the variable. I wull update it with my next PR, in which I am making changes in the FT and NFT DAOs.